Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM node:6 AS builder
+
+RUN groupadd --gid 1100 lisk && \
+    useradd --create-home --home-dir /home/lisk --shell /bin/bash --uid 1100 --gid 1100 lisk
+COPY . /home/lisk/lisk/
+RUN chown lisk:lisk --recursive /home/lisk/lisk
+
+USER lisk
+WORKDIR /home/lisk/lisk
+
+RUN npm install
+RUN ./node_modules/.bin/grunt release
+
+
+FROM node:6
+
+ENV CONFD_VERSION 0.16.0
+ENV CONFD_SHA256 255d2559f3824dd64df059bdc533fd6b697c070db603c76aaf8d1d5e6b0cc334
+
+RUN groupadd --gid 1100 lisk && \
+    useradd --create-home --home-dir /home/lisk --shell /bin/bash --uid 1100 --gid 1100 lisk
+COPY --from=builder /home/lisk/lisk/ /home/lisk/lisk/
+COPY docker_files/ /
+# git repository needed for build; cannot added to .dockerignore
+RUN rm -rf /home/lisk/lisk/.git && \
+    mkdir /home/lisk/lisk/logs && \
+    chown lisk:lisk --recursive /home/lisk/lisk
+
+RUN curl --location --output /tmp/confd \
+         https://github.com/kelseyhightower/confd/releases/download/v${CONFD_VERSION}/confd-${CONFD_VERSION}-linux-amd64 && \
+    if [ x"$( sha256sum /tmp/confd |awk '{ print $1 }' )" = x"${CONFD_SHA256}" ]; then \
+        mv /tmp/confd /usr/local/bin/; \
+	chmod +x /usr/local/bin/confd; \
+    else \
+        rm -f /tmp/confd; \
+	exit 1; \
+    fi
+
+ARG LISK_VERSION
+ARG LISK_MIN_VERSION=${LISK_VERSION}
+ENV LISK_VERSION ${LISK_VERSION}
+ENV LISK_MIN_VERSION ${LISK_MIN_VERSION}
+
```

Comment:
Is there a reason you are not adding the version specific seed peers here using 
```
ENV LISK_PEERS_LIST_1 "94.237.41.99:5001"

ENV LISK_PEERS_LIST_2 "209.50.52.217:5001"

ENV LISK_PEERS_LIST_3 "94.237.26.150:5001"

ENV LISK_PEERS_LIST_4 "83.136.249.102:5001"

ENV LISK_PEERS_LIST_5 "94.237.65.179:5001"
```
Like here: https://github.com/LiskHQ/lisk-docker/commit/c8b49ff761fd8980d08808c299ebc9e158bbbd8c#diff-ed9d43ac74c2d8e77f8497b190a5ab5aR23

For ease of release purposes the version specific nethash and seed peers should be set in this list, shouldn't they ?

Reply:
If we add see nodes here then this `Dockerfile` becomes specific to a network (betanet given the peers you listed)

Reply:
Ah I thought you might want to do this because there are other network specific files on the repo and to maintain compatibility with the old Docker images. But since the Docker image is using the "next" version they are breaking anyway and the proposed adding of an extra file for the peers list is a very good idea.